airbyte-webapp/src/config/configProviders.test.ts

```diff
@@ -29,7 +29,7 @@ describe("applyProviders", function () {
         innerProp: "1",
       },
     };
-    const providers: Provider<DeepPartial<Value>>[] = [
+    const providers: ProviderAsync<DeepPartial<Value>>[] = [
```

Comment:
:information_source: Fixed some broken type, that apparently slipped previous TS versions.